devito/propagator.py

```diff
@@ -480,6 +483,48 @@ def generate_space_loops_blocking(self, loop_body):
 
         return [loop_body] + full_remainder if full_remainder else [loop_body]
 
+    def get_number_of_loads(self):
+        """Gets number of loads in the kernel"""
+        return len(self._kernel_dic_oi["load_all_list"])
```

Comment:
Asked @vincepandolfo and he confirmed that it still exists and `Profiler` does not provide that info